PLATFORM_README.md

```diff
@@ -127,24 +127,49 @@ For an existing Angular CLI application,
 
     For models prior to 0.11.1 use `fundamental-ngx`
 
+    ```javascript
     import { SwitchModule } from '@fundamental-ngx/core';
 
     @NgModule({
     ...
     imports: [SwitchModule],
     })
     export class DemoModule { }
+    ```
+
+3. **Add the component to your HTML.**
 
+    ```html
+    <fd-switch [size]="'l'" [(checked)]="myValue">Large Switch</fd-switch>
     ```
 
+4. **To add support for i18n**
+
+    To mark a text for translation, use [Angular's i18n marker](https://angular.io/guide/i18n#mark-text-with-the-i18n-attribute).
+
+    For example, to translate the button text, add `i18n` on the element `fdp-button`:
+
+    ```html
+    <fdp-button i18n="@@emphasizedBtn" [type]="'emphasized'">Emphasized Button</fdp-button>
     ```
 
-3. **Add the component to your HTML.**
+    It is recommended to give ID to the marker like `@@emphasizedBtn` for easier tracking in the extracted `messages.xlf` file.
+
+    Note: Some components allow you to provide data imperatively or through an array of objects like `[list]="seasons"` in the below example:
```

Comment:
I would remove the "or" in this sentence:
> Some components allow you to provide data imperatively ~~or~~ through an array of objects like `[list]="seasons"` in the below example

Reply:
done